MetaMorpheus/TaskLayer/CalibrationTask/CalibrationTask.cs

```diff
@@ -189,7 +184,24 @@ protected override MyTaskResults RunSpecific(string OutputFolder, List<DbForTask
 
             return MyTaskResults;
         }
+        private bool CalibrationHasValue(DataPointAquisitionResults acquisitionResultsFirst, DataPointAquisitionResults acquisitionResultsSecond)
+        {
+            bool improvedCounts = acquisitionResultsSecond.Psms.Count > acquisitionResultsFirst.Psms.Count && acquisitionResultsSecond.Psms.Select(p => p.FullSequence).Distinct().Count() > acquisitionResultsFirst.Psms.Select(p => p.FullSequence).Distinct().Count();
```

Comment:
Should probably only calculate the peptide counts once and set to local variable to avoid performing the select and distinct twice

Reply:
done